test/unit/geo/transform.test.js

```diff
@@ -231,6 +231,66 @@ test('transform', (t) => {
         t.end();
     });
 
+    t.test('getBounds (#10261)', (t) => {
```

Comment:
It would be nice to have a test  for `_getBounds3D` with transform isolated, but it requires `elevation.visibleDemTiles`. Perhaps we could create a mock for that?

Reply:
I think that should work by mocking it in such a way that you get a valid `minmax` value, as it's the only dependency on `elevation` in this method.